plugins/importexport/medra/MedraExportPlugin.inc.php

```diff
@@ -102,11 +103,21 @@ function depositXML($objects, $context, $filename) {
 		assert(is_readable($filename));
 		$xml = file_get_contents($filename);
 		assert($xml !== false && !empty($xml));
+		
+		// Select the language
+		$lang = 'eng';
+		if(substr($this->_request->getUserVar('language'), 0, 2) == 'it'){
```

Comment:
This is intended to be the choosen language of the OJS user, because the result of the xml file validation when deposit to Crossref should be display in the language of the user CMS OJS. Do you mean that with the submission's locale?

Reply:
We've moved away from letting the user's current locale define what kind of metadata is exported in other plugins (see e.g. https://github.com/pkp/jatsTemplate/commit/1b99ed185bb6b49555757e0e8a5e9a1fb3c8fdfe) -- I think exports/deposits should always be created independent of what the user's UI language is.